components/scream/src/share/util/units.hpp

```diff
@@ -51,7 +51,9 @@ class Units {
   // Construct a non-dimensional quantity
   Units (const ScalingFactor& scaling)
    : m_scaling {scaling}
-   , m_units{0,0,0,0,0,0,0}
+   , m_units{RationalConstant(0), RationalConstant(0), RationalConstant(0),
```

Comment:
Did you try this:

```
, m_units{ {0}, {0}, ... }
```

Reply:
I think Jim's solution may work.

Reply:
Just tried that. Same error. This is probably one of those things they put in before it was fully baked. :-/